src/admin/systemTypes/systemTypes.component.tsx

```diff
@@ -31,6 +33,27 @@ interface TableRowData extends SystemType {
   isSpare: boolean;
 }
 
+function isExactFilterActive(
+  table: MRT_TableInstance<TableRowData>,
+  expectedFilters: { id: string; filterFn?: string; value: string }[]
+) {
+  const actualFilters = table.getState().columnFilters;
+  const actualFilterFns = table.getState().columnFilterFns;
```

Comment:
```suggestion
  const { columnFilters, columnFilterFns } = table.getState();
```
So that state is only called once, bit more readable as well

Reply:
I'll change it, but i don't see how it is more readable. 

`const { columnFilters: actualFilters, columnFilterFns:actualFilterFns } = table.getState();`